src/Parser.js

```diff
@@ -515,34 +504,6 @@ export default class Parser {
                     endNameToken);
             }
             return result;
-        } else if (func in Parser.oldFontFuncs) {
-            const style = Parser.oldFontFuncs[func];
-            // If we see an old font function, parse out the implicit body
-            this.consumeSpaces();
-            const body = this.parseExpression(true, breakOnTokenText);
-            if (style.slice(0, 4) === 'text') {
-                return new ParseNode("text", {
-                    style: style,
-                    body: new ParseNode("ordgroup", body, this.mode),
-                }, this.mode);
```

Comment:
This block was never executed b/c there were no values that started with `text` in `oldFontFuncs`.